javaslang/src/main/java/javaslang/collection/SortedMap.java

```diff
@@ -147,6 +148,12 @@
     <K2, V2> SortedMap<K2, V2> map(BiFunction<? super K, ? super V, Tuple2<K2, V2>> mapper);
 
     @Override
+    <K2> SortedMap<K2, V> mapKeys(Function<? super K, ? extends K2> keyMapper);
+
+    @Override
+    <K2> SortedMap<K2, V> mapKeys(Function<? super K, ? extends K2> keyMapper, Function2<V, V, V> valueMerge);
```

Comment:
Also `BiFunction<? super V, ? super V, ? extends V>` instead of `Function<V, V, V>`